client/components/tinymce/index.jsx

```diff
@@ -195,8 +195,16 @@ module.exports = React.createClass( {
 	},
 
 	componentDidMount: function() {
+		this.mounted = true;
+
 		const setup = function( editor ) {
```

Comment:
I find it's easier to read if you move function definitions out of other functions. It's a nice to have and not a blocker :)